.env.example

```
@@ -99,3 +99,6 @@ RAID_JIRA_PROJECT_KEY="T2"
 FF_SLACK_SKIP_CHECKS=true
 # Disable SSO redirect for local dev by setting to true. When SSO is disabled, go to /admin/ to login
 FF_DEBUG_NO_SSO_REDIRECT=false
+
+## TOOLBOX settings
+TOOLBOX_URL=https://toolbox.mycompany.com/login
```

```suggestion
RAID_TOOLBOX_URL=https://toolbox.mycompany.com/login
```
Put in the `RAID settings` block line 93